src/pretix/control/logdisplay.py

```diff
@@ -421,6 +423,36 @@ def display(self, logentry: LogEntry, data):
         )
 
 
+@log_entry_types.new_from_dict({
+    "pretix.event.order.data_sync.success": _("Ticket data successfully transferred to {provider}."),
+})
+class OrderDataSyncLogentrytype(OrderLogEntryType):
```

Comment:
Logentrytype or LogEntryType? Let's stay consistent (same for other class below)

Reply:
```suggestion
class OrderDataSyncLogEntryType(OrderLogEntryType):
```